MultiPage-Clone/css/main.css

```diff
@@ -1,3 +1,528 @@
+  flex-direction: column;
+  align-items: center;
+  height: fit-content;
+}
+
+.background-index {
+  background: linear-gradient(
+    180deg,
+    var(--White) 270px,
+    var(--Dark-beige) 270px
+  );
+}
+
+.background-dark {
+  background: var(--Dark-beige);
+}
+
+.card-section {
+  background-color: var(--Light-beige);
+  width: 88vw;
+  height: fit-content;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+.shadow {
+  box-shadow: var(--shd, 0 1px 4px rgba(0, 0, 0, 0.6));
+}
+.personal-information {
+  height: fit-content;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 40px 0px;
+  gap: 30px;
+}
+.my-img {
+  width: 50%;
+  border-radius: 9999px;
+}
+
+.card {
+  max-width: 6ch;
+  text-align: center;
+}
+
+.blue-line {
+  background-color: var(--Royal-blue);
+  width: 20%;
+  height: 3px;
+  border-radius: 9999px;
+}
+.button {
+  display: flex;
+  flex-direction: row;
+  gap: 20px;
+  font-size: 28px;
+}
+
+.btn-card {
+  width: 150px;
+  height: 45px;
+  border-radius: 9999px;
+  border: none;
+  background-color: var(--Royal-blue);
+  color: var(--White);
+  font-weight: 600;
+}
+.empty {
+  border: 1.5px solid var(--Black);
+  color: var(--Black);
+  background-color: var(--Royal-blue), 0;
+}
+
+.empty:hover {
+  background-color: var(--Royal-blue);
+}
+.media-information {
+  height: 50px;
+  background-color: var(--White);
+  margin: 0px 0px 20px;
+  display: flex;
+  flex-direction: row;
+  gap: 20px;
+  align-items: center;
+  justify-content: center;
+}
+
+.media-icon {
+  width: 25px;
+}
+
+.section-p {
+  width: 80vw;
+  margin: 45px;
+}
+.main-header {
+  margin: 45px 0px 15px;
+  font: var(--font-header1);
+  font-size: 55px;
+}
+.second-hdr {
+  font: var(--font-header1);
+  font-size: 22px;
+  font-weight: 600;
+}
+.sec-information {
+  font: var(--font-header1);
+  font-weight: 300;
+  font-size: 18px;
+  margin: 20px 0px;
+}
+
+.section-cont {
+  padding-top: 9em;
+}
+.left-side-btn {
+  display: none;
+}
+
+.ftr-media-icon {
+  width: 15px;
+  margin-bottom: 20px;
+}
+
+@media screen and (min-width: 540px) {
+  main {
+    margin-top: 120px;
+    flex-direction: row;
+    justify-content: center;
+  }
+  .background-index {
+    background: linear-gradient(
+      90deg,
+      var(--Dark-beige) 40vw,
+      var(--White) 40vw
+    );
+    min-height: calc(100vh - 240px);
+  }
+  .background-dark {
+    min-height: calc(100vh - 240px);
+  }
+
+  .card-section {
+    width: 375px;
+  }
+
+  .section-p {
+    max-width: 375px;
+  }
+
+  .left-side-btn {
+    display: inline;
+  }
+  .side-btn {
+    display: none;
+  }
+
+  .second-hdr {
+    margin: 30px 0px;
+  }
+
+  .sec-information {
+    margin: 30px 0px;
+  }
+
+  .main-header {
+    font-size: 100px;
+  }
+
+  .in-card::before {
+    display: none;
+  }
+
+  .media-information {
+    margin-left: 40px;
+  }
+
+  .nav-links {
+    font-size: 1.5em;
+  }
+
+  .nav-links1 {
```

Comment:
when clicking on the nav, there is no animation on your page, and the text is much smaller and takes up less of the page!